docs/default_base_image.md

```diff
@@ -0,0 +1,13 @@
+# Default Base Images in Jib
+
+## Jib Build Plugins 3.0
+
+Starting from version 3.0, the default base image is the official [`adoptopenjdk`](https://hub.docker.com/_/adoptopenjdk) image on Docker Hub. AdoptOpenJDK (which is [being renamed to Adoptium](https://blog.adoptopenjdk.net/2020/06/adoptopenjdk-to-join-the-eclipse-foundation/)) is a popular OpenJDK build also used by [Google Cloud Buildpacks](https://github.com/GoogleCloudPlatform/buildpacks) (for Java) by default. Prior to 3.0, Jib Maven and Gradle plugins built images based on Distroless Java when the user didn't make an explicit choice. The Jib team carefully assessed all the surrounding factors and ultimately decided to switch from Distroless for the best interest of the Jib users. That is, to meet one of the core missions of Jib to build a secure and optimized image.
+
+Note that this change matters only when you are not specifying any base image via `jib.from.image` (Gradle) or `<from><image>` (Maven), which we have recommended against consistently. For strong reproducibility, you should pin to a specific base image using a digest (or at least a tag). And we want to be clear that Jib's default choice for AdoptOpenJDK does not imply any endorsement to it; you should do your due diligence when making a choice for a base image.
```

Comment:
Hm can we elaborate a bit more on why we recommend against specifying the base image?  It could cause some confusion because we also mention later on that the change doesn't imply endorsement for the new default and users have a choice to pick another base image.

Reply:
Oh, maybe the opening sentence made you misunderstand. It's that we have consistently recommended specifying a base image (to pin to a specific image), not _against_. I'll try rewording it.

Reply:
Oh! That makes sense! Sorry, totally my bad. I think the double negatives may have confused me.